### PR TITLE
Update readme to include official docs link

### PR DESCRIPTION
**Pull request by bisht2050**
Adding link to the official docs and retiring links to the text files in the repo.

### Checklist

- [ ] Add tests and ensure they pass
- [ ] Add an entry to the CHANGELOG.md file
- [ ] Update documentation for new features

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,13 +12,8 @@ This package was renamed to `mongodb/laravel-mongodb` because of a transfer of o
...
 
-- [Installation](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/install/)
-- [Eloquent Models](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/eloquent-models/)
-- [Query Builder](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/query-builder/)
-- [Transactions](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/transactions/)
-- [User Authentication](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/user-authentication/)
-- [Queues](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/queues/)
-- [Upgrading](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/upgrade/)
+## Documentation
+Reference documentation for the Laravel MongoDB package is available [here](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/).
```
Comment (GromNaN) [inline README.md]: Link on "here" text is not recommended. 

Maybe something like this (not sure):
```suggestion
Read the reference documentation for the Laravel MongoDB package [on MongoDB website](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/).
```

**Review by GromNaN — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (bisht2050) [reply README.md]: Thanks, makes sense.

File: README.md [hunk 2]
```diff
@@ -12,13 +12,8 @@ This package was renamed to `mongodb/laravel-mongodb` because of a transfer of o
...
 
-- [Installation](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/install/)
-- [Eloquent Models](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/eloquent-models/)
-- [Query Builder](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/query-builder/)
-- [Transactions](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/transactions/)
-- [User Authentication](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/user-authentication/)
-- [Queues](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/queues/)
-- [Upgrading](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/upgrade/)
+## Documentation
+Read the reference documentation for the Laravel MongoDB package [on MongoDB website](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/).
```
Comment (jmikola) [inline README.md]: I'd suggest copying the syntax we already use in https://github.com/mongodb/mongo-php-library/blob/master/README.md#documentation

```md
## Documentation

 - https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/
 - https://www.mongodb.com/docs/drivers/php/
```

Also, I just noticed that https://mongodb.com/docs/ecosystem/drivers/php/ redirects to https://www.mongodb.com/docs/drivers/php/. I'll make that change to the PHPLIB README.

File: README.md [hunk 3]
```diff
@@ -12,13 +12,9 @@ This package was renamed to `mongodb/laravel-mongodb` because of a transfer of o
...
 
-- [Installation](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/install/)
-- [Eloquent Models](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/eloquent-models/)
-- [Query Builder](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/query-builder/)
-- [Transactions](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/transactions/)
-- [User Authentication](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/user-authentication/)
-- [Queues](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/queues/)
-- [Upgrading](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/upgrade/)
+## Documentation
+- https://www.mongodb.com/docs/drivers/php/laravel-mongodb/
```
Comment (jmikola) [inline README.md:17]:
```suggestion

- https://www.mongodb.com/docs/drivers/php/laravel-mongodb/
```

Line break, consistent with other sections.

**Review by GromNaN — APPROVED** [review]

**Pull request merged**